src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int AMOUNT_BALLS = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after class declaration or method signature. Remove the empty line at 7.